CHANGELOG.md

```diff
@@ -12,6 +12,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - added new method `Grid::delete_orders` and the corresponding switch
   `--delete-orders` in the subcommand `write` of the CLI
 
+### Changed
+
+- renamed `pineappl.lumi.rs` to `pineappl.channel.rs`
+- removed `.create()` and `.from_grid()` methods and directly instantiate objecs from the class
```

Comment:
Removed items should be listed under a `### Removed` heading

Reply:
```suggestion
- removed `Grid.create()` and `FkTable.from_grid()` methods; use the constructors of the respective classes instead
```
+ I suggest a slight rewording